Source/Exceptions/AxoDefaultException.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <exception>
+
+class AxoDefaultException : public std::exception
+{
+public:
+	AxoDefaultException(const std::string& message);
+	~AxoDefaultException() override = default;
+
+	const char* what() const noexcept override;
```

Comment:
```suggestion
	const char* What() const noexcept override;
```

should not this be What()?

Reply:
Notice that this is an override, we're extending std::exception: https://en.cppreference.com/w/cpp/error/exception/what

Reply:
okay, now I understand it. I didn't see that override. thank you!